src/main/java/com/datadoghq/connect/logs/sink/DatadogLogsSinkConnectorConfig.java

```diff
@@ -34,13 +34,14 @@ This product includes software developed at Datadog (https://www.datadoghq.com/)
 
     // Only for testing
     public Boolean useSSL = true;
+    public String url = "http-intake.logs.datadoghq.com:443";
```

Comment:
At least in a bigger project, it is usually not a good idea to make public fields non final for testing.
Someone can accidentally used the field outside unit test as it find it convenient. A common solution to handle this scenario is to have one parameterless constructor which set the default values (`useSSL = true` and `url = "http-intake.logs.datadoghq.com:443"`) and another constructor that takes `url` and `useSSL` as parameter.